Solutions/SlashNext SIEM/Data/Solution_SlashNextSIEM.json

```diff
@@ -1,7 +1,7 @@
 {
     "Name" : "SlashNext SIEM",
     "Author" : "SlashNext - support@slashnext.net",
-    "Logo" : "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/SlashNext%20SIEM/Playbook/logo/slashnext-logo.svg\" width=\"75px\" height=\"75px\">",
+    "Logo" : '<img src="https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/SlashNext%20SIEM/Playbook/logo/slashnext-logo.svg" width=\"75px\" height=\"75px\">',
```

Comment:
Hello @SaadatAbid-snx, Please replace the above logo link with "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/SlashNext%20SIEM/Playbook/logo/slashnext-logo.svg\"

Reply:
@v-prasadboke  you mean I should change it to "<img src="https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/SlashNext%20SIEM/Playbook/logo/slashnext-logo.svg" width=\"75px\" height=\"75px\">"
but in double quotes, I can not use double quotes again so we need an escape character **\"** so it will be like
"<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/SlashNext%20SIEM/Playbook/logo/slashnext-logo.svg" width=\"75px\" height=\"75px\">"
Kindly re-assure if I am wrong.

Reply:
Hello @HamailSaleemi-snx, If possible can you please provide me access to your branch.

Reply:
"<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/SlashNext%20SIEM/Playbook/logo/slashnext-logo.svg\" width="75px" height="75px">"

Please copy this and paste it within Inputfile of data folder.